dev-libs/spdlog/files/spdlog-unbundle-fmt.patch

```diff
@@ -1,90 +1,37 @@
-diff --git a/CMakeLists.txt b/CMakeLists.txt
-index 52d2953..d132d26 100644
---- a/CMakeLists.txt
-+++ b/CMakeLists.txt
-@@ -80,6 +80,7 @@ install(
- install(
-     DIRECTORY "${HEADER_BASE}/${PROJECT_NAME}"
-     DESTINATION "${include_install_dir}"
-+    PATTERN "bundled*" EXCLUDE
- )
- 
- # install project version file
 diff --git a/include/spdlog/fmt/fmt.h b/include/spdlog/fmt/fmt.h
-index 92ca4e5..55f1fa8 100644
+index e2e0485..d33bc45 100644
 --- a/include/spdlog/fmt/fmt.h
 +++ b/include/spdlog/fmt/fmt.h
-@@ -10,25 +10,7 @@
- // By default spdlog include its own copy.
- //
- 
--#if !defined(SPDLOG_FMT_EXTERNAL)
--
--#ifndef FMT_HEADER_ONLY
--#define FMT_HEADER_ONLY
--#endif
--#ifndef FMT_USE_WINDOWS_H
--#define FMT_USE_WINDOWS_H 0
--#endif
+@@ -18,16 +18,11 @@
+ #ifndef FMT_USE_WINDOWS_H
+ #define FMT_USE_WINDOWS_H 0
+ #endif
 -#include "bundled/format.h"
 -#if defined(SPDLOG_FMT_PRINTF)
 -#include "bundled/printf.h"
 -#endif
--
--#else //external fmtlib
--
+ 
+-#else // external fmtlib
++#endif // external fmtlib
```

Comment:
The 'external fmtlib' comment is technically wrong now, since 'external fmtlib' code is no longer inside `#ifdef`.